mlsshared.sty

```diff
@@ -197,6 +200,8 @@
   belowskip=0pt,
   defaultdialect=[MLS]C,
 }
+% Note: Needing size for sans serif font, or LaTeXML will have different size for them which looks weird.
```

Comment:
I think we might as well put the explanation on the `commentstyle=` line instead of the silly `% comment style`.  There, we can explain why we must repeat the same font size selection as in `basicstyle` (which is what I don't understand at the moment).

Reply:
Moved.